recipes/jwt-cpp/all/test_package/conanfile.py

```diff
@@ -6,6 +6,9 @@ class JsonCppTestConan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
     generators = "cmake", "cmake_find_package"
 
+    def requirements(self):
+        self.requires("picojson/1.3.0")
```

Comment:
I think it should depend on package options.

Reply:
But it works with nearly any JSON library what would the option be?

Reply:
I see, you need at least one json lib, I guess.

Reply:
Yes just for the test_package